src/data/languages/Spanish.json

```diff
@@ -0,0 +1,444 @@
+{
+	"info": {
+		"language": "Español",
+		"language_english": "Spanish",
+		"fallback": ["English"],
+		"maintainer": "MrJako2001",
+		"maintainer_link": "https://anilist.co/user/MrJaco/",
+		"discussion_link": "",
+		"notes": "This is the default language of Automail. This key should have all the translation keys, while other translations don't have to. I'm not a native speaker, but I try to stick to British English here (feel free to britpick!). You can make translation files for other version of English too if you want, name it something like 'English (US)', and put 'English' in the fallback array. You need only include the keys that are different."

```

Comment:
That's the notes for the default English language file, leave blank "" if you have no notes specific to Spanish.

Reply:
Fixed